### PR TITLE
Added Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by pedrominatel**
Added the Brazilian Portuguese (pt-BR) translation to the installer.

### CODE REVIEW

**Comment by pedrominatel** [comment]
Hi, @georgik can you add some Brazilian colleague as reviewer?

File: src/InnoSetup/Languages/IdfToolsSetup_pt-BR.islu [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+SystemCheckApplyFixesButtonCaption=Aplicar Correções
+SystemCheckFullLogButtonCaption=Log completo
+SystemCheckApplyFixesConsent=Você quer aplicar os comandos com as correções sugeridas, a fim de atualizar o seu ambiente Windows e iniciar uma nova checagem do sistema?
+SystemCheckFixesSuccessful=Correções aplicadas com sucesso.
+SystemCheckFixesFailed=Aplicação das correções falhou. Por favor veja o log completo para maiores informações.
+SystemCheckNotCompleteConsent=Checagem do sistema não concluída. Deseja prosseguir sem a checagem do sistema?
+SystemCheckRootCertificates=Verificando certificados
+SystemCheckRootCertificateWarning=Não foi possível carregar dados do servidor dl.espressif.com.
+SystemCheckForLongPathsEnabled=Verificando registro "LongPathsEnabled" no sistema
+SystemCheckRemedyFailedLongPathsEnabled=Por favor alterar o registro HKLM\SYSTEM\CurrentControlSet\Control\FileSystem\LongPathsEnabled para 1. The operation requires Administrator privileges. Command:
```
Comment (euripedesrocha) [inline src/InnoSetup/Languages/IdfToolsSetup_pt-BR.islu]: The operation requires Administrator privileges -> A operação requer privilégios de Administrador

Comment (pedrominatel) [reply src/InnoSetup/Languages/IdfToolsSetup_pt-BR.islu]: Done!

**Comment by georgik** [comment]
Thank you @pedrominatel for the contribution.

**Pull request merged**